### ISSUE TITLE
doc build errors on mac 10.10.3

### ISSUE THREAD

**Issue by doutriaux1** [nco/nco#10]
```
Making all in doc
restore=: && backupdir=".am$$" && \
	am__cwd=`pwd` && CDPATH="${ZSH_VERSION+.}:" && cd . && \
	rm -rf $backupdir && mkdir $backupdir && \
	if (/bin/sh /git/nco/autobld/missing makeinfo --version) >/dev/null 2>&1; then \
	  for f in nco.info nco.info-[0-9] nco.info-[0-9][0-9] nco.i[0-9] nco.i[0-9][0-9]; do \
	    if test -f $f; then mv $f $backupdir; restore=mv; else :; fi; \
	  done; \
	else :; fi && \
	cd "$am__cwd"; \
	if /bin/sh /git/nco/autobld/missing makeinfo   -I . \
	 -o nco.info nco.texi; \
	then \
	  rc=0; \
	  CDPATH="${ZSH_VERSION+.}:" && cd .; \
	else \
	  rc=$?; \
	  CDPATH="${ZSH_VERSION+.}:" && cd . && \
	  $restore $backupdir/* `echo "./nco.info" | sed 's|[^/]*$||'`; \
	fi; \
	rm -rf $backupdir; exit $rc
nco.texi:7139: warning: unlikely character ( in @var.
nco.texi:7139: warning: unlikely character ) in @var.
nco.texi:19499: Unknown command `textdegree'.
nco.texi:19499: Misplaced {.
nco.texi:19499: Misplaced }.
nco.texi:19500: Unknown command `textdegree'.
nco.texi:19500: Misplaced {.
nco.texi:19500: Misplaced }.
nco.texi:19500: Unknown command `textdegree'.
nco.texi:19500: Misplaced {.
nco.texi:19500: Misplaced }.
nco.texi:19501: Unknown command `textdegree'.
nco.texi:19501: Misplaced {.
nco.texi:19501: Misplaced }.
nco.texi:19502: Unknown command `textdegree'.
nco.texi:19502: Misplaced {.
nco.texi:19502: Misplaced }.
nco.texi:19503: Unknown command `textdegree'.
nco.texi:19503: Misplaced {.
nco.texi:19503: Misplaced }.
nco.texi:19503: Unknown command `textdegree'.
nco.texi:19503: Misplaced {.
nco.texi:19503: Misplaced }.
nco.texi:19504: Unknown command `textdegree'.
nco.texi:19504: Misplaced {.
nco.texi:19504: Misplaced }.
makeinfo: Removing output file `nco.info' due to errors; use --force to preserve.
make[2]: *** [nco.info] Error 1
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
```
4.5.0 tag point and master

**Comment by czender** [comment]
Yes, I had forgotten about this. Thanks for the reminder. It's due to the extremely old Texinfo tools that ship with the Mac. The offending symbol is in makeinfo/Texinfo 4.12 but MacOS must still ship something older (4.8?). This is one reason I use MacPorts, which has Texinfo 5.2. I guess the long -term solution is to abort the doc build unless Texinfo >= 4.12. Will try to do this for 4.5.1.The interim workaround is to install a newer makeinfo/texinfo.

**Comment by doutriaux1** [comment]
thanks, or just turn off the doc build (that's what I did)

**Comment by czender** [comment]
how do you turn-off the doc-build? i can put this in the FAQ for other mac users...

**Comment by doutriaux1** [comment]
I hack the "Makefile" by hand :wink: 
```
SUBDIRS = data src man doc
```
become
```
SUBDIRS = data src man
```

**Comment by czender** [comment]
This was presumably solved in NCO 4.5.3 release with new option configure --disable-doc

**Comment by lukecampbell** [comment]
Also I was able to fix this issue by installing texinfo with homebrew

```
brew install texinfo
brew link -f texinfo
```
